NEWS

```diff
@@ -295,8 +297,8 @@ CHANGES WITH 256 in spe:
         Systemd-boot and systemd-stub and Related Tools:
 
         * TPM 1.2 PCR measurement support has been removed from systemd-stub.
-          TPM 1.2 is obsolete and — due to the (by today's standards) weak
-          cryptographic algorithms it only supports — does not actually provide
+          TPM 1.2 is obsolete and—due to the (by today's standards) weak
+          cryptographic algorithms it only supports—does not actually provide
```

Comment:
Please don't do this. I know it's the usual English style, but it makes no sense. It's pretty bad with a real font, where the long dash can be made wide, but it's even worse with a monospace font where the long dash is displayed as a normal dash. NEWS is supposed to be readable with a monospace font.